doc/whats_new/v0.14.rst

```diff
@@ -45,7 +45,7 @@ Changelog
 - Ability to pass one penalty (alpha value) per target in
   :class:`linear_model.Ridge`, by @eickenberg and `Mathieu Blondel`_.
 
-- Fixed :mod:`sklearn.linear_model.stochastic_gradient.py` L2 regularization
```

Comment:
I'm not sure we should be fixing the old whats_new (I haven't), but not critical